pkg/features/features.go

```diff
@@ -15,7 +15,7 @@ type FeatureFlag interface {
 	Name() string
 	EnvVar() string
 	Enabled() bool
-	Default() bool
+	Released() bool
```

Comment:
Maybe not in this PR but could you add comments to describe what's the difference between `Released` and `Enabled`

Reply:
#12122